### PR TITLE
Add Slovak translations

### PR DESCRIPTION
**Pull request by romana-la**
## Summary

Translation of the documentation into Slovak language.

## Related Issues

Is part of #738

## Checklist

- [x] This pull request has a meaningful title and description.
- [x] I have read and followed the [Contributing guidelines](https://github.com/danielschuster-muc/potter-db/blob/master/CONTRIBUTING.md).
- [x] My code follows the conventions and coding style of this project.
- [x] All tests pass and I have added tests for my changes 
- [x] I have updated necessary documentation

### CODE REVIEW

File: docs/pages/index.sk.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Vitajte v dokumentácii k **Potter DB: API**!
+Vstúpte do magického sveta Harryho Pottera a získajte informácie o postavách, knihách, zaklínadlách a ďalšie dáta.
+Či už ste developer vyvíjajúci aplikáciu týkajúcu sa Harryho Pottera alebo len zvedavý fanúšik, táto dokumentácia vás prevedie tajmi nášho API.
+
+## Čo je to Potter DB: API?
+Potter DB: API je REST a GraphQL rozhranie pre rozsiahly repozitár dát z magického sveta Harryho Pottera. My sa staráme o dáta, zatiaľ čo vy sa môžete sústrediť na vývoj vašej aplikácie. Táto API vám umožní:
+- získať detailné informácie o vašich obľúbených postavách (`characters`) vrátane informácií o ich pozadí a fakulte
+- získať ucelený prehľad zaklínadiel (`spells`) a odvarov (`potions`) z čarodejníckeho sveta
+- získať podrobné informácie o knihách (`books`) a filmoch (`movies`) zo sveta Harryho Pottera
```
Comment (danielschuster-muc) [inline docs/pages/index.sk.mdx:86]: It's nice that you added the english translation / key of the resources here.

File: docs/pages/resources.sk.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Card, Cards } from 'nextra/components'
+
+# Prehľad zdrojov
+
+<Cards>
+  <Card
+    title="Knihy (books)"
```
Comment (danielschuster-muc) [inline docs/pages/resources.sk.mdx:7]: I am not quite sure if the english translations are needed here.

Comment (danielschuster-muc) [reply docs/pages/resources.sk.mdx:7]: I mean it's not bad to have, but what do you think, is it needed to understand it?

File: docs/pages/resources/_meta.sk.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "books": "Knihy (books)",
+  "chapters": "Kapitoly (chapters)",
+  "characters": "Postavy (characters)",
+  "movies": "Filmy (movies)",
+  "potions": "Odvary (potions)",
+  "spells": "Zaklínadlá (spells)"
+}
```
Comment (danielschuster-muc) [inline docs/pages/resources/_meta.sk.json:8]: same here

File: docs/pages/resources/chapters.sk.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Kapitoly (chapters)
```
Comment (danielschuster-muc) [inline docs/pages/resources/chapters.sk.mdx:1]: also here and on the other pages, do we really need the english translation of the resource keys?

File: docs/pages/apis/graphql.sk.mdx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+# GraphQL API
+Naše GraphQL API ponúka flexibilný a efektívny spôsob interakcie s Potter DB. Táto sekcia vám predstaví kľúčové aspekty nášho GraphQL API a vy sa dozviete, ako ho využiť vo svoj prospech.
+
+## Koncový bod GraphQL
+
+Na získanie a zmenu dát cez GraphQL API pošlite požiadavku POST koncovému bodu GraphQL (`/graphql`). Telo požiadavky musí obsahovať váš dopyt a premenné a musí byť vo formáte JSON.
+
+## Dopytovanie dát
+Vďaka GraphQL môžete získať presne tie dáta, ktoré potrebujete. Dopyty musia obsahovať polia, o ktoré máte záujem, a vzťahy medzi nimi. Napríklad:
```
Comment (danielschuster-muc) [inline docs/pages/apis/graphql.sk.mdx:15]:
```suggestion
## Dopytovanie dát

Vďaka GraphQL môžete získať presne tie dáta, ktoré potrebujete. Dopyty musia obsahovať polia, o ktoré máte záujem, a vzťahy medzi nimi. Napríklad:
```

File: docs/pages/apis/graphql.sk.mdx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  }
+  character(slug: "harry-potter") {
+    name
+  }
+}
+```
+V tomto dopyte chceme získať názov (`title`) a dátum vydania (`releaseDate`) prvých troch kníh, ako aj názov (`title`) a slug (`slug`) každej kapitoly vo všetkých troch knihách. Okrem toho chceme získať aj meno (`name`) postavy so slugom `harry-potter`.
+
+## Schéma
+Kompletnú schému GraphQL API nájdete [tu](https://github.com/danielschuster-muc/potter-db/blob/master/backend/app/graphql/schema.json).
```
Comment (danielschuster-muc) [inline docs/pages/apis/graphql.sk.mdx:47]:
```suggestion
## Schéma

Kompletnú schému GraphQL API nájdete [tu](https://github.com/danielschuster-muc/potter-db/blob/master/backend/app/graphql/schema.json).
```

File: docs/pages/apis/rest.sk.mdx [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+ID musí byť vo formáte UUID alebo slug.
+
+</details>
+
+## Špecifikácia podľa štandardu OpenAPI (OAS)
+
+Na uľahčenie integrácie poskytujeme špecifikáciu v súlade s [OpenAPI](https://openapis.org) štandardom, ktorá popisuje jednotlivé koncové body API, dátové modely a schémy požiadaviek a odpovedí. Naše API sa riadi verziou 3.0.3 OpenAPI štandardu.
+Aktuálnu API špecifikáciu nájdete [tu](https://api.potterdb.com/v1/openapi.json).
+
+## Formát JSON:API
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.sk.mdx:43]:
```suggestion
## Formát JSON:API

```

File: docs/pages/getting-started.sk.mdx [hunk 8]
```diff
@@ -0,0 +1,42 @@
+import { Bleed } from 'nextra-theme-docs'
+
+# REST vs. GraphQL
```
Comment (danielschuster-muc) [inline docs/pages/getting-started.sk.mdx:3]:
```suggestion
# REST vs. GraphQL

```

File: docs/pages/getting-started.sk.mdx [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+    </tr>
+    <tr>
+      <td>Aktuálna verzia</td>
+      <td>v1</td>
+    </tr>
+  </tbody>
+</table>
+}
+
+## Obmedzenie maximálneho počtu požiadaviek
```
Comment (danielschuster-muc) [inline docs/pages/getting-started.sk.mdx:36]:
```suggestion
## Obmedzenie maximálneho počtu požiadaviek

```

File: docs/pages/resources/chapters.sk.mdx (hunk 4 above)
Comment (danielschuster-muc) [inline docs/pages/resources/chapters.sk.mdx:1]:
```suggestion
# Kapitoly (chapters)

```

File: docs/pages/resources/potions.sk.mdx [hunk 10]
```diff
@@ -0,0 +1,99 @@
+# Odvary (potions)
```
Comment (danielschuster-muc) [inline docs/pages/resources/potions.sk.mdx:1]:
```suggestion
# Odvary (potions)

```

File: docs/pages/resources/spells.sk.mdx [hunk 11]
```diff
@@ -0,0 +1,87 @@
+# Zaklínadlá (spells)
```
Comment (danielschuster-muc) [inline docs/pages/resources/spells.sk.mdx:1]:
```suggestion
# Zaklínadlá (spells)

```

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Thanks for your contribution! Looks pretty good so far. Just some smaller things, see my comments.

**Comment by danielschuster-muc** [comment]
Hi @romana-la, are there any updates from your side? Could you please resolve the requested changed, so I can merge you pr?

**Comment by romana-la** [comment]
Hi @danielschuster-muc, thanks for your feedback and sorry that it took me so long to get to it. I have implemented most of your suggestions. Re. having resources in both English and Slovak - I have dropped English from the statistics table but I would still keep it in the table of contents and titles. When somebody makes a request, they will have to specify the resource in the request URL in English. When they then look in the documentation for the explanation of attributes to interpret the response, they will be likely looking for the English word.

**Comment by danielschuster-muc** [comment]
> Hi @danielschuster-muc, thanks for your feedback and sorry that it took me so long to get to it. I have implemented most of your suggestions. Re. having resources in both English and Slovak - I have dropped English from the statistics table but I would still keep it in the table of contents and titles. When somebody makes a request, they will have to specify the resource in the request URL in English. When they then look in the documentation for the explanation of attributes to interpret the response, they will be likely looking for the English word.

Yes, you definitely have a point, makes sense to have the resources names in English and Slovak. Thank you for your input.

**Review by danielschuster-muc — APPROVED** [review]
Looks very nice! Thanks for your contribution.

**Comment by kolja-lucht** [comment]
Hey @danielschuster-muc 
could provide the German translation 🧙‍♀️

**Comment by danielschuster-muc** [comment]
> Hey @danielschuster-muc could provide the German translation 🧙‍♀️

sure! maybe you can leave a short note in #738

**Comment by kolja-lucht** [comment]
Oops, I thought I had commented on that issue. Didn't realize I was in a PR 😅 Sorry about that @danielschuster-muc

**Pull request merged**